### PR TITLE
Fix failing tests

### PR DESCRIPTION
**Pull request by strategio**
I the first commit I actually fixed the tests: Some changes were added in the node configuration and even though the
tests were changed, it has been lost (maybe a rebase).

See those commits:
- 890cc69310335ad5e2e678d4beef019ec14ca8b2
- 9792718623782d01b751146e3cda018b5c4d4f96
- 6cd73b728264c946d02f3b6babe1bae702e75b41

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-6114

I added a second commit to move the node configuration to a public static function. To me, there's no point to test the configuration (that is, to copy the configuration into the tests).

### CODE REVIEW

**Comment by kagg-design** [comment]
+1 @strategio

**Review by kagg-design — APPROVED** [review]
+1

**Comment by strategio** [comment]
@OnTheGoSystems/wpml please check the second commit.

File: src/class-wpml-elementor-translatable-nodes.php [hunk 1]
```diff
@@ -172,11 +173,10 @@ private function conditions_ok( $node_data, $element ) {
 		return $conditions_meet;
 	}
 
-	public function initialize_nodes_to_translate() {
-
-		$this->nodes_to_translate = array(
+	public static function get_nodes_to_translate() {
```
Comment (andreasciamanna) [inline src/class-wpml-elementor-translatable-nodes.php:176]: @strategio why changing it to static? We don't use static methods here

**Review by andreasciamanna — CHANGES_REQUESTED** [review]

File: src/class-wpml-elementor-translatable-nodes.php (hunk 1 above)
Comment (strategio) [reply src/class-wpml-elementor-translatable-nodes.php:176]: @andreasciamanna, the returned array does not depends on the state of the object. It's more a configuration array.

Comment (andreasciamanna) [reply src/class-wpml-elementor-translatable-nodes.php:176]: But all classes implementing `IWPML_Page_Builders_Translatable_Nodes` use non-satic method for `initialize_nodes_to_translate`.

The interface itself does not declare the method as static.

Declaring it static only in `class-wpml-elementor-translatable-nodes.php` not only makes the implementation inconsistent but violates the interface's contract.

Comment (strategio) [reply src/class-wpml-elementor-translatable-nodes.php:176]: @andreasciamanna, I didn't remove `initialize_nodes_to_translate`, it's still here https://github.com/OnTheGoSystems/wpml-page-builders-elementor/pull/52/files#diff-1e5046e82faeba27c319819ef7a502caR673.

Comment (andreasciamanna) [reply src/class-wpml-elementor-translatable-nodes.php:176]: I've confused one method with the other. Sorry.

**Review by andreasciamanna — APPROVED** [review]

**Pull request merged**